### PR TITLE
Avoid ppi<10 causing UI to break

### PR DESCRIPTION
**Pull request by cdrini**
Testing: https://www-drini.archive.org/details/micro_IAMFTEST00_0022/page/60/mode/1up now works

### CODE REVIEW

File: src/BookReader/BookModel.js [hunk 1]
```diff
@@ -352,8 +352,8 @@ export class PageModel {
    * @param {PageIndex} index
    */
   constructor(book, index) {
-    // TODO: Get default from config
-    this.ppi = book._getDataProp(index, 'ppi', book.ppi);
+    // Values less than 10 cause the UI to not work correctly
+    this.ppi = Math.max(book._getDataProp(index, 'ppi', book.ppi), 10);
```
Comment (cdrini) [inline src/BookReader/BookModel.js:356]:
```suggestion
    // Values less than 10 cause the UI to not work correctly
    const pagePPI = book._getDataProp(index, 'ppi', book.ppi);
    this.ppi = Math.max(pagePPI < 10 ? book.ppi : pagePPI, 10);
```

File: src/BookReader/BookModel.js [hunk 2]
```diff
@@ -352,8 +352,9 @@ export class PageModel {
    * @param {PageIndex} index
    */
   constructor(book, index) {
-    // TODO: Get default from config
-    this.ppi = book._getDataProp(index, 'ppi', book.ppi);
+    // Values less than 10 cause the UI to not work correctly
+    const pagePPI = book._getDataProp(index, 'ppi', book.ppi);
+    this.ppi = Math.max(pagePPI < 10 ? book.ppi : pagePPI, 10);
```
Comment (iisa) [inline src/BookReader/BookModel.js:357]: 👍 q: what does pagePPI default to?

**Review by iisa — APPROVED** [review]
🚢

File: src/BookReader/BookModel.js (hunk 2 above)
Comment (cdrini) [reply src/BookReader/BookModel.js:357]: Oh the default is the last argument to getDataProp, book.ppi . This is the value in the metadata for that entire book

**Pull request merged**